data/rabbit/image/rubykaigi2024-images/property.rb

```diff
@@ -0,0 +1,4 @@
+@category = N_("Image")
+@title = N_("RubyKaigi2024Image")
+@abstract = N_("Images related to RubyKaigi2024")
+@description = N_("Provides images related to RubyKaigi2024.")
```

Comment:
I think that `rubykaigi-2024` is better but I used `rubykaigi2011`...
https://github.com/rabbit-shocker/rabbit/tree/master/lib/rabbit/theme/rubykaigi2011

Anyway, we can use `rubykaigi-2024` because the "RubyKaigi 2024" (has a space) is the official name. And we should mention license somewhere. 

```suggestion
@title = N_("RubyKaigi 2024 image")
@abstract = N_("Images related to RubyKaigi 2024")
@description = N_("Provides images related to RubyKaigi 2024. They are licensed under CC BY 4.0 by RubyKaigi 2024 Team.")
```